lib/packages.js

```diff
@@ -82,8 +82,12 @@ api.readPackagesSync = rootModule => {
   // sort packages according to dependencies
   _packageCache = _sortPackages(_packageCache);
 
-  Object.keys(_packageCache).forEach(
-    name => _getPackageOverrides(_packageCache[name]));
+  // manifest override must happen here to be applied to compile-less`
```

Comment:
Do we need to do manifest override prior to sorting? Couldn't it affect sort order? I would think so. I wonder if we actually need two passes: sort, apply manifest overrides, sort again, later apply package overrides. That would allow sorting to be changed based on manifest overrides but ensure that manifest overrides are applied based on "original" package sort order.